openshift/performance/scripts/run-all-performance-tests.sh

```diff
@@ -0,0 +1,238 @@
+  oc label nodes "${without_activator[0]}" knative-ingress=true
+  without_ingress=("${without_activator[@]:1}")
+
+  oc adm taint nodes "${without_ingress[0]}" serverless=true:NoSchedule
+  oc adm taint nodes "${without_ingress[1]}" serverless=true:NoSchedule
+
+  oc label nodes "${without_ingress[0]}" serverless=true
+  oc label nodes "${without_ingress[1]}" serverless=true
+
+  oc patch knativeserving knative-serving \
+    -n "${SYSTEM_NAMESPACE}" \
+    --type merge --patch '{"spec":{"workloads":[{"name":"activator","replicas":2,"resources":[{"container":"activator","requests":{"cpu":"500m","memory":"1000Mi"},"limits":{"cpu":"2000m","memory":"4000Mi"}}],"tolerations":[{"key":"knative-activator","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"knative-activator":"true"}},{"name":"controller","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"domain-mapping","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"domainmapping-webhook","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"net-kourier-controller","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"autoscaler","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"webhook","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"autoscaler-hpa","tolerations":[{"key":"serverless","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"serverless":"true"}},{"name":"3scale-kourier-gateway","tolerations":[{"key":"knative-ingress","operator":"Equal","effect":"NoSchedule","value":"true"}],"nodeSelector":{"knative-ingress":"true"}}]}}'
+
+  # enforce all pods to be re-scheduled on the new nodes only
+  oc delete po --all -n knative-serving --force --grace-period=0
+  oc delete po --all -n knative-serving-ingress --force --grace-period=0
+
+  # make sure we are in good shape after re-scheduling and patching worked as expected
+  oc wait --for=condition=Ready knativeserving.operator.knative.dev knative-serving -n "${SYSTEM_NAMESPACE}" --timeout=900s
+
+else
+  header "Scaling cluster"
+  for name in $(oc get machineset -n openshift-machine-api -o name); do oc scale "$name" -n openshift-machine-api --replicas=4; done
+  oc wait --for=jsonpath="{.status.availableReplicas}=4" machineset --all -n openshift-machine-api --timeout=-1s
+
+  oc patch knativeserving knative-serving \
+    -n "${SYSTEM_NAMESPACE}" \
+    --type merge --patch '{"metadata": {"annotations": {"serverless.openshift.io/default-enable-http2": "true" }}}'
+fi
+
+sleep 100
```

Comment:
What's the goal of this sleep?

Reply:
Yeah probably it is a relic, I will remove, when run manually it didn't help.